### ISSUE TITLE
Refactor Frontend System Configuration to Opensource 

### ISSUE THREAD

**Issue by simonemainardi** [ntop/ntopng#4714]
Currently, frontend system configuration is in

```
simone@apu2:~/ntopng$ less pro/scripts/lua/nedge/system_setup/
captive_portal.lua   dhcp.lua             gateways.lua         misc.lua             network.lua          security.lua         shapers.lua
date_time.lua        dns.lua              interfaces.lua       mode.lua             routing.lua     static_routes.lua
```

It is requested to move to the opensource part  `/system_setup/` of ntopng:

- misc.lua
- network.lua
- security.lua
- date_time.lua
- dns.lua 
- interfaces.lua
- mode.lua

and leave the rest in the nEdge source code. Basically, the opensource part of ntopng will allow only configuration of the system and interfaces

**Comment by cardigliano** [comment]
An object oriented backend for system setup (split into nf_config and appliance_config) has been implemented in ba9c501c4dd65d3e836bfed1fae72c3f80e7967a , also frontend lua files to be used in "appliance" mode have been moved to the opensource part as requested. 
(The appliance mode implementation is in progress)

**Comment by cardigliano** [comment]
Migration has been completed (working on wifi and applications configuration now as next step)
